theme/static/js/resourceLandingAjax.js

```diff
@@ -195,14 +195,14 @@ function metadata_update_ajax_submit(form_id){
                 // file type 'coverage' element gets updated for composite resource
                 if ((json_response.element_name.toLowerCase() === 'site' && resourceType === 'Time Series') ||
                     ((json_response.element_name.toLowerCase() === 'coverage' ||
-                    json_response.element_name.toLowerCase() === 'site') && resourceType === 'Composite Resource')){
+                    json_response.element_name.toLowerCase() === 'site') && resourceType === 'Resource')){
                     if (json_response.hasOwnProperty('temporal_coverage')){
                         var temporalCoverage = json_response.temporal_coverage;
                         updateResourceTemporalCoverage(temporalCoverage);
                         // show/hide delete option for resource temporal coverage
                         setResourceTemporalCoverageDeleteOption();
 
-                        if(resourceType === 'Composite Resource' && json_response.has_logical_temporal_coverage) {
```

Comment:
does this "Composite Resource" need a change?
https://github.com/hydroshare/hydroshare/blob/4853-resource-type-name-fix/hs_core/management/commands/set_aggregations_dirty.py#L55

Reply:
I think change is not needed since this is in a management command user doesn't see it.